apps/epic-react/src/components/lesson-completion-toggle.tsx

```diff
@@ -7,10 +7,14 @@ import {trpcSkillLessons} from '@skillrecordings/skill-lesson/utils/trpc-skill-l
 import toast from 'react-hot-toast'
 
 const LessonCompleteToggle = () => {
-  const {module} = useLesson()
+  const {module, lesson} = useLesson()
+  const flattenedLessons = module.sections?.flatMap(
+    (section) => section.lessons,
+  )
+  const currentLessonIndex =
+    flattenedLessons?.findIndex((l) => l?.slug === lesson.slug) ?? 0
```

Comment:
There are top-level lessons that aren't within a section, so I'm thinking `module.sections.flatMap` will miss those lessons.

`module.lessons` might be all of them for a given module

Reply:
ah that's what this was for 

https://github.com/skillrecordings/products/pull/1605#discussion_r1664788713

Reply:
my only hesitance with this is that there might be some skill-stack internals that expects `module.sections` to only contain sections, I'm not certain tho. I just remember having to tread carefully when I was getting the Collection List display to work with ER shaped data.